### PR TITLE
Compare grid

### PR DESCRIPTION
**Pull request by jakelauer27**
Created two components for the compare grid functionality.

1. The compare class builds the compare characters table and the column headers.
It also houses the sort functions for sorting the grid.

2. The compareGridItem function that builds a row for each character

### CODE REVIEW

**Comment by JustinTorrance** [comment]
Great job dynamically creating elements and inserting data! Your sorting logic looks sound as well.

File: public/Images/universe_icons/flame_smash_bros.svg [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) by Toomai Glittershine for SmashWiki -->
+<svg id="svg2" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:cc="http://creativecommons.org/ns#" xmlns:dc="http://purl.org/dc/elements/1.1/"
```
Comment (libbyeh) [inline public/Images/universe_icons/flame_smash_bros.svg]: Yes, love  this "flame" fill. I think we should use on all instances of the smash-logo. Great choice!

File: public/index.html [hunk 2]
```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet">
```
Comment (libbyeh) [inline public/index.html]: Thank you for adding the font family to the html. No edits (unless we decide on an additional font later on)

File: src/Compare.js [hunk 3]
```diff
@@ -26,6 +27,9 @@ class Compare extends Component {
     e.target.classList.remove('up')
```
Comment (libbyeh) [inline src/Compare.js]: I believe the only addition to this was the carats next to the name for filtering. I see that you added a conditional for add, sort and remove

File: src/Compare.js [hunk 4]
```diff
@@ -69,69 +76,56 @@ class Compare extends Component {
   render() {
     return (
       <div className='compare-page'>
```
Comment (libbyeh) [inline src/Compare.js]: Thank you for walking me through this section, I am clear on what it is rendering and it looks really clean.

No edits.

File: src/CompareItem.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './styles/App.css';
+
+const CompareItem = (props) => {
```
Comment (libbyeh) [inline src/CompareItem.js]: Tied to the compare.js file (render section), I am clear on the code below and don't have any changes.

File: src/styles/App.css [hunk 6]
```diff
@@ -1,3 +1,42 @@
+table, th, tr, td {
```
Comment (libbyeh) [inline src/styles/App.css]: Coode looks great for the compare styling

**Pull request merged**